### PR TITLE
General clean up of the code, added proper logging library

### PR DESCRIPTION
**Pull request by CheatCod**

### CODE REVIEW

File: src/managers/instance_manager.rs [hunk 1]
```diff
@@ -35,7 +33,12 @@ impl InstanceManager {
         let mut instance_collection: HashMap<String, ServerInstance> = HashMap::new();
         let mut instance_list = vec![];
         let mut taken_ports = HashSet::new();
-        let mut tunnel_manager = TunnelManager::new(path.join(".lodestone").join("bin").join("frpc"), "server_properties".to_owned(), 8001, Some("test".to_owned()));
+        let mut tunnel_manager = TunnelManager::new(
```
Comment (YZmarkS) [inline src/managers/instance_manager.rs:36]: kevin look through this

**Review by Lemonsity — COMMENTED** [review]
reviewed those I can

instance_manager, macro_manager, and server_isntances changed too much, cannot say much

File: src/managers/instance_manager.rs (hunk 1 above)
Comment (CheatCod) [reply src/managers/instance_manager.rs:36]: this module is not complete yet

**Review by Ynng — APPROVED** [review]
lgtm!

**Pull request merged**